src/studio/nuxeo-studio/working-in-studio/templates.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Templates
+review:
+    comment: ''
+    date: '2018-10-25'
+    status: ok
+tree_item_index: 650
+toc: true
+---
+## Document Templates
+
+Document templates are used as conversion instructions, to render XML, HTML, CSV or any open format from a document or a list of documents, using specific rendering operations. You would especially want to use this when you handle business data on content types and want to generate a nicer presentation for publishing, export, etc.
+
+For more information on Document Templates, discover [how to quickly generate a PDF using document template]({{page version='' space='nxdoc' page='how-to-quickly-generate-a-pdf-using-document-template'}}).
```

Comment:
You can also add that the Nuxeo Template Rendering addon is leveraging the templating mechanism